provision_docker_server.sh

```diff
@@ -1,6 +1,6 @@
 sudo apt-get remove docker docker-engine docker.io
 sudo apt-get update
-sudo apt-get install \
+sudo apt-get install -y \
```

Comment:
This seems unrelated to this PR but is a good change. I would consider either having these apt-get calls use `-qy` to be less noisy and maybe setting `DEBIAN_FRONTEND="noninteractive"` in the environment. If there's a chance that this will run on systems which have been upgraded from older versions of Docker it might also be a good idea to have the `install` calls use `-o Dpkg::Options::='--force-confnew'` to have APT overwrite any common config files even if there are local changes so you're always starting from the package's default config.